README.md

```diff
@@ -1 +1,206 @@
-# Capstone-project
\ No newline at end of file
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Festival_Amani_Website] <a name="about-project"></a>
+
+**[Festival_Amani_Website]** is a project that will allow people to what the festival amani provide in Goma town and the date of the next session of the Amani festival.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com">Html</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://CSS.com/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Client</summary>
+  <ul>
+    <li><a href="https://www.javascript.com">Javascript</a></li>
+  </ul>
+</details>
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217464405-3193d403-f400-4311-b115-598196912a59.png)

- [ ] You have three Clients which is not making any sense :thinking: Kindly remove two of them to make your readme file professional and descriptive :+1: